worlds/stardew_valley/logic/logic.py

```diff
@@ -281,7 +281,7 @@ def __init__(self, player: int, options: StardewValleyOptions, content: StardewC
             Material.coal: self.mine.can_mine_in_the_mines_floor_41_80() | self.tool.has_tool(Tool.pan),
             Material.fiber: True_(),
             Material.hardwood: self.tool.has_tool(Tool.axe, ToolMaterial.copper) & (self.region.can_reach(Region.secret_woods) | self.region.can_reach(Region.island_west)),
-            Material.moss: True_(),
+            Material.moss: self.season.has_any_not_winter(),
```

Comment:
Can we also add access to forest and scythe/weapon for future proofing?

Reply:
You can get it from trees on the farm, yes, but we do not have a logic rule for "can grow trees" (which is **not** sphere 1), so pretty much all logic rules that require trees have been using the forest, where trees grow naturally, as a placeholder for it. Trees also grow in the mountain, bus stop, railroad and a few island maps, but the forest has the most trees

Reply:
As for the tool, yes, any tool except watering can works. But scythe and sword can do it for free (no energy cost), so they are generally preferred. Plus, the axe harms the tree permanently, and since you need a very old tree to get moss, using the axe every time significantly reduces your long-term moss obtention rate as you'd kill the tree every few harvests. i'd be fine with the rule being any tool, I just thought scythe/sword would be nicer